src/app-logic.js

```diff
@@ -58,30 +33,31 @@ export default function useAppLogic() {
 
         totalActiveTokens = totalActiveTokens.plus(totalActive)
 
-        const myStake = proposal.stakes.find(
-          stake => stake.entity === connectedAccount
+        const myStake = proposal.stakes.find(stake =>
+          addressesEqual(stake.entity, account)
         )
 
         if (myStake) {
           myStakes.push({
-            proposal: proposal.id,
+            proposalId: proposal.id,
             proposalName: proposal.name,
-            stakedAmount: myStake.amount,
+            amount: myStake.amount,
```

Comment:
Note that decided to update this attributes to be consistent across all the app's data